data_steward/bq_utils.py

```diff
@@ -87,21 +87,7 @@ def get_retraction_submission_folder():
 
 
 def get_combined_deid_dataset_id():
-    """
-    Get the combined unidentified dataset id.
-
-    If the environment variable has not been set, default to the defined name,
-    set the environment variable for the process, and return the dataset name.
-
-    :return:  A name for the combined unidentified dataset id.
-    """
-    dataset_id = os.environ.get(bq_consts.COMBINED_DEID_DATASET,
-                                bq_consts.BLANK)
-    if dataset_id == bq_consts.BLANK:
-        date_string = datetime.now().strftime(bq_consts.DATE_FORMAT)
-        dataset_id = bq_consts.COMBINED_DEID_DATASET_FORMAT.format(date_string)
-        os.environ[bq_consts.COMBINED_DEID_DATASET] = dataset_id
-    return dataset_id
+    return os.environ.get('COMBINED_DEID_DATASET_ID')
```

Comment:
hey @lrwb-aou I advised @ksdkalluri to remove this logic based on what you and I discussed yesterday; could you confirm this is ok? I don't want to break anything relying on this and I couldn't find other usages other than `clean_cdr.clean_combined_de_identified_dataset`.

Reply:
@lrwb-aou @mark-velez sorry for replying late regarding this, I assumed that removing that logic will affect any of the participant validation which is my bad.

Reply:
@ksdkalluri  and @mark-velez   This looks good.  It shouldn't affect anything other than the `clean_ehr_rdr_de_identified_dataset` function in `clean_cdr.clean_combined_de_identified_dataset`.  It will behave just like everything else in `clean_cdr.py`.  That's good.

The only issue I see is, in `clean_cdr.py` we are logging that a dataset was unspecified and we are falling back to a default value.  This is fine if we have a default value to return from `bq_utils`.  What happens to all these functions if the environment variable isn't set?  This is a separate issue.  It should be handled in a separate PR.  We should add a check to make sure our dataset_id is not `None` after attempting to set it through `bq_utils`.

Reply:
Thanks @lrwb-aou . I added to the next Engs Sync agenda some time to discuss unset env vars and defaults. I think this discussion will help us formulate an issue ticket.